sources/US/UT/rockymountaingoat.json

```diff
@@ -1,15 +1,17 @@
 {
-   "name": "UT Mountain Goat units",
-   "url": "http://dwrcdc.nr.utah.gov/ucdc/DownloadGIS/Data/Abiotic/2016_RMGoat.zip",
```

Comment:
This data looks like it is versions yearly, so the thing to do is leave the existing file, and make a new one for 2017 with 2017 in the filename, and put year in the name.